rest-api-spec/src/main/resources/rest-api-spec/test/indices.shrink/30_copy_settings.yml

```diff
@@ -1,8 +1,6 @@
 ---
 "Copy settings during shrink index":
   - skip:
```

Comment:
pre this removal would this skip section be applied to versions `- 6.9.99` **AND** `features warnings` ?
or **OR** ?
my point is, should we now remove the whole section?

Reply:
Ah good catch, looks like the test doesn't throw any actual warnings so the skip isn't needed. :+1: